modules/task_collection.js

```diff
@@ -58,7 +58,41 @@ class TaskCollection {
         button.index = index;
         button.ref = ref;
       }, ref);
+      const checks = document.querySelectorAll('.action_check');
+      checks.forEach((check,index) =>{
+
+        check.addEventListener('click',(event) =>{
```

Comment:
- [OPTIONAL] Nice implementation, but according to the project specification, it would to nice to use a `change event listener` instead of a `click event listener`.

![eventListener](https://user-images.githubusercontent.com/65852621/155526912-86e597a7-bf6f-47b6-bd5e-e2a2a9a080cd.png)